src/index.js

```diff
@@ -42,7 +44,7 @@ app.use(
 );
 // CONNECTING APP TO POSTGRESS
 
-const sequelize = new Sequelize(configurations.env_configurations.development.url, {
+const sequelize = new Sequelize(development.url, {
   dialect: "postgres"
 });
 
```

Comment:
You don't this connection anymore. Sequelize has already taken care of it in the models/index.js file